.github/workflows/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Install dependencies
         if: steps.cache.outputs.cache-hit != 'true'
-        run: yarn
+        run: yarn && yarn dev:prepare
```

Comment:
This should be pnpm now no? (with `corepack enable` step before)
```suggestion
        run: pnpm enable && pnpm install && pnpm dev:prepare
```

Reply:
Thanks! Updated this script.